db/migrate/20160820205817_create_publishers.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePublishers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :publications, id: :uuid  do |t|
+      t.string :name
+      t.string :type
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
I tried that but the migration file name reverted back on the migrate, unless I did something funky.  I think it's some rails magic that ensures you don't fuck with migrations.